core/src/main/java/org/mule/runtime/core/api/el/ExpressionLanguage.java

```diff
@@ -22,6 +22,13 @@
 public interface ExpressionLanguage {
 
   /**
+   * Registers the given {@link BindingContext} as global.
```

Comment:
Can you add javadoc about what would happen with a previous global context if another one is registered? Or if once a global context was registered it can/cannot be changed?. What about concurrency against possible expressions under execution?